.github/workflows/generated-linux-binary-manywheel-nightly.yml

```diff
@@ -165,11 +165,11 @@ jobs:
       DESIRED_CUDA: cu117
       GPU_ARCH_VERSION: 11.7
       GPU_ARCH_TYPE: cuda
-      DOCKER_IMAGE: pytorch/manylinux-builder:cuda11.7
+      DOCKER_IMAGE: tousif111/manylinux-builder:cuda11.7
```

Comment:
Docker image location needs to stay pytorch/manylinux-builder:cuda11.7

Reply:
Same as above. It's a temporary change. The docker image is built from https://github.com/pytorch/builder/pull/1196 and pushed to my personal docker hub.